src/index.html

```diff
@@ -0,0 +1,282 @@
+          <p class="discover-card__body">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua invidunt ut labore. </p>
+        </div>
+        <div class="discover">
+          <div class="discover__item">
+            <img class="discover__icon" src="/image/sales.png" alt="">
+            <div class="discover__detail">
+              <h1 class="discover__title">Sales Tracking</h1>
+<p class="discover__body">
+  Lorem ipsum dolor sit amet, consetetur sadipscing elitr.
+            </p>
+            </div>
+            
+          </div>
+          <div class="discover__item">
+          <img class="discover__icon" src="/image/project.png" alt="">
+            <div class="discover__detail">
+              <h1 class="discover__title">Project Management</h1>
+<p class="discove-card__body">
+  Lorem ipsum dolor sit amet, consetetur sadipscing elitr.
+            </p>
+            </div>
+          </div>
+          <div class="discover__item">
+           <img class="discover__icon" src="/image/activity.png" alt="">
+            <div class="discover__body">
+              <h1 class="discover__title">Activity Report</h1>
+<p class="discover__body">
+  Lorem ipsum dolor sit amet, consetetur sadipscing elitr.
+            </p>
+            </div>
+          </div>
+         
+        </div>
+      </div>
+      <div class="features">
+        <h1 class="features__title">Features</h1>
+        <div class="features-section flex-section--features">
+     <div class=" features-card">
+       <img class="features-card__icon" src="/image/encryption.svg" alt="">
+       <h1 class="features-card__title">Advanced 256-bit encryption
+      </h1>
+      <p class="features-card__body">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor.</p>
+     </div>
+     <img src="/image/Divider.svg" alt="">
+     <div class="features-card">
+       <img class="features-card__icon" src="/image/tools.svg" alt="">
+       <h1 class="features-card__title">Simple collaboration tools
+      </h1>
+      <p class="features-card__body">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor.</p>
+    </div>
+    <img src="/image/Divider.svg" alt="">
+     <div class="features-card">
+       <img class="features-card__icon" src="/image/ai.svg" alt="">
+       <h1 class="features-card__title">Customizable AI features
+      </h1>
+      <p class="features-card__body">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor.</p>
+     </div>
+        </div>
+        </div>
+      <div class="testimonials">
+        <h1 class="testimonials__title">The stunning results our customers have experienced</h1>
+        <div class="testimonials-items">
+          <div class="testimonial-card">
```

Comment:
I would name this `testimonial__card` instead, and `testimonials-items` would be `testimaonials__items` since they are contained within the `testimonials` block